src/ILLink.CodeFix/ILLink.CodeFixProvider.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is this the common setup? I'd expect the analyzer and fix to live in the same assembly so they don't get accidentally out of sync

Reply:
They can't -- the fixer is an IDE tool, while the analyzer is a compiler plugin. The environment for the two is different, and the workspaces layer (that the fixer depends on) doesn't exist in the compiler.

Reply:
Do we plan to ship this or is it just for our use? I'm wondering how we would package it to avoid including two copies of the analyzer dll in the sdk. The SDK looks like it uses a single input for both analyzers and code fixes: https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Analyzers.targets#L111

Reply:
I think we should probably ship this. I'll have a follow-up PR for the NuGet authoring for that.